daffodil-tdml-lib/src/test/scala/org/apache/daffodil/tdml/UnitTestTDMLRunner.scala

```diff
@@ -345,6 +345,46 @@ class UnitTestTDMLRunner {
     assertEquals(expected, actual)
   }
 
+  @Test def testCommentBit(): Unit = {
```

Comment:
Corner case to add a test for:

```
<documentPart type="bits">
// this doc part contains no bits 
// at all. It is just comments.
// 101010101
</documentPart>
```

Also, I think this should not be an error:
```
<documentPart type="bits">01011010 // just a comment here no line ending </documentPart>
```
We should put a test in for this. 
The reason I like this is that it lets short things stay on a single line.